### DISCUSSION TITLE
How to create a custom activity

### DISCUSSION THREAD

**Discussion by Nao050101** [elsa-workflows/elsa-guides#17] [Q&A]
I would like to know about custom activities for Elsa workflow.
I was able to set it up by following the instructions below.
https://elsa-workflows.github.io/elsa-core/docs/next/quickstarts/quickstarts-elsa-and-docker

I want to add my own activity to elsa workflow.
It would be helpful to have a sample project, as I could not understand it from the following only.
It would be great if you could tell me.
https://elsa-workflows.github.io/elsa-core/docs/next/extensibility/extensibility-custom-activities

**Comment by manasheed** [comment]
Hi @Nao050101 ,
I am following this tutorial. It may help you too.

https://medium.com/codex/part-6-of-building-workflow-driven-net-applications-with-elsa-2-9b3167c612dd

**Reply by asaadhoussam** [reply]
When I using the designer to create new workflow, I don't see the custom activities that were added in workflow/activities in elsa server
can you help me for this issue?
